docs/eigenlayer/operator-guides/key-management/institutional-operators.md

```diff
@@ -6,5 +6,10 @@ sidebar_position: 2
 
 - Secure keys, including secrets such as passphrases or mnemonics, using services like AWS Secrets Manager or Hashicorp Vault. These services can be seamlessly integrated with automated mechanisms that safely retrieve secrets or keys (e.g., remote signers). If resources permit, consider running your own Hashicorp Vault instance, which grants full custody of keys and secrets while sacrificing the service provider's availability and security guarantees.
 - Avoid generating all keys with the same mnemonic. Minimize the attack surface by employing a new mnemonic for every 200 or 1000 validator keys, depending on your preference. This approach also reduces the risk of losing key generation capabilities if a single mnemonic is lost, and limits the impact if an attacker gains access to a few mnemonics.
+- Given that Avs keys are likely to be much fewer, not using a seed to generate the keys is probably safer; generate each avs key independently if possible.
 - Use a remote signer like **[Web3signer](https://github.com/ConsenSys/web3signer)** or, better yet, distributed signers to eliminate single points of failure.
-- Develop a custom solution involving tailor-made tools. For instance, use Web3signer for remote signing and store keys on AWS Secrets Manager. A custom tool can manage automatic key storage in Secrets Manager and facilitate interactions with Web3signer.
\ No newline at end of file
+- Develop a custom solution involving tailor-made tools. For instance, use Web3signer for remote signing and store keys on AWS Secrets Manager. A custom tool can manage automatic key storage in Secrets Manager and facilitate interactions with Web3signer.
+
+# Smart Contract Operators
+
+We encourage institutional operators to register with eigenlayer using an [erc-1271](https://eips.ethereum.org/EIPS/eip-1271) smart contract wallet. This allows a lot more fine-grained control, such as multisig authorization and key rotation, which is currently not possible for EOA operators.
```

Comment:
Added this as a starting template but someone with better knowledge should expand on this.

Reply:
cc: @nelsonijih and @stevennevins

Reply:
Would this include a Safe contract? Could we add any example smart contract wallets?

Reply:
lgtm

Reply:
@wesfloyd good question. 
@shrimalmadhur can we link to the example 1271 contracts you wrote for puffer or something?
@stevennevins do safe contracts abide to 1271 interface? If not how are they related to 1271 contracts? Are there any oeprators using safes right now? If so, can we add another section explaining this as well.

Reply:
I didn't write a contract. they wrote a contract which we could refer once they make it public - I just wrote a helper function to get eigenda registration parameters